app/javascript/entrypoints/application.js

```
@@ -0,0 +1,39 @@
+import * as ReactRailsUJS from "react_ujs";
+
+// Settings
+import "../settings";
+
+import "../stylesheets/application.scss";
+
+Rails.start();
+ActiveStorage.start();
+
+// Support component names relative to this directory:
+const componentRequireContext = import.meta.glob('../src/components/**/*.{jsx,tsx}', { eager: true });
```

ESLint

[error] Replace `'../src/components/**/*.{jsx,tsx}',·{·eager:·true·}` with `⏎··"../src/components/**/*.{jsx,tsx}",⏎··{·eager:·true·}⏎`

(prettier/prettier)